.github/dependabot.yml

```
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "./view/next-project"
+    schedule:
+      interval: "weekly"
+    groups:
```

```suggestion
    schedule:
      interval: "weekly"
    versioning-strategy: increase
    groups:
```

> Dependabot が作成するプルリク[エス](https://d.hatena.ne.jp/keyword/%A5%A8%A5%B9)ト上で対象ライブラリがpackage.jsonとpackage-lock.jsonの両方に存在するにも関わらずpackage-lock.jsonしか更新してくれない事象が発生していました。
> 
> これは Depedabot の設定によるもので、dependabot.ymlから設定できるversioning-strategyをincreaseにすることで弊チームの環境では解消しました。

これ入れといた方が良いらしい。
https://tech.visasq.com/dependabot-to-handle-vulnerabilities

---

ちょっとよくわからんから後で教えて欲しいかも

versioning-strategy: increase
新しいバージョン出るとすぐ更新されそう

---

> versioning-strategy: increase
> 新しいバージョン出るとすぐ更新されそう

確かに
でもpackageもすぐ更新するではない？

---

口頭で解決